### PR TITLE
README.md edited

### PR DESCRIPTION
**Pull request by emach039**
@jessb0t Changes to the readme file! I'll be adding the new members eventually once they're completely onboarded and working on the project :)

### CODE REVIEW

**Comment by jessb0t** [comment]
Thank you so much for updating this readme, @emach039 !  I made some line-by-line suggestions!

**Review by jessb0t — CHANGES_REQUESTED** [review]

**Review by emach039 — COMMENTED** [review]
Will make necessary edits!

File: README.md [hunk 1]
```diff
@@ -1,33 +1,46 @@
...
 
 
 ## Roadmap
-:point_right: Details of each data release should be documented in this readme. 
 
+**Data collection**: anticipated Q3 2021 to Q2 2022
 
-## Work in Development
-This `main` branch contains completed releases for this project. For all work-in-progress, please switch over to the `dev` branches.
+**Data analysis and processing** (Release 1): anticipated Q2 2022
```
[the remark this answers is not in the document]
Comment (emach039) [reply README.md:19]: Noted!

File: README.md [hunk 2]
```diff
@@ -1,33 +1,46 @@
...
 
-:point_right: Keep the "Work in Development" text above as the default NDCLab "blurb" to help visitors navigate the repo. But delete this note before publishing the readme file.
+**Paper ready-for-submission** (Release 2): anticipated Q3 2022
```
[the remark this answers is not in the document]
Comment (emach039) [reply README.md:21]: Noted!

**Comment by jessb0t** [comment]
@emach039 It looks like you integrated these changes over on the `dev-contributing` branch, so I'm going to proceed with merging this branch in (and then changes from `dev-contributing` will overwrite).

**Review by jessb0t — APPROVED** [review]

**Pull request merged**